src/screens/AccountNetworkChooser.js

```diff
@@ -50,32 +50,34 @@ class AccountNetworkChooserView extends React.PureComponent {
     return (
       <ScrollView style={styles.body} contentContainerStyle={{ padding: 20 }}>
         <Text style={styles.title}>CHOOSE NETWORK</Text>
-        { Object.entries(NETWORK_LIST).map(([networkKey, networkParams]) => (
-          <TouchableItem
-            key={networkKey}
-            style={[
-              styles.card,
-              {
-                marginTop: 20,
-                backgroundColor: networkParams.color
-              }
-            ]}
-            onPress={() => {
-              accounts.updateNew(empty('', networkKey));
-              navigation.goBack();
-            }}
-          >
-            <Text
+        { Object.entries(NETWORK_LIST)
+          .filter(([networkKey]) => networkKey !== UnKnownNetworkKeys.UNKNOWN )
```

Comment:
The commit looks bigger than it is, all I did is add this filter to make sure `Unkown network` isn't showing up.